src/terra-dev-site/about/Features.b.about.mdx

```diff
@@ -25,7 +25,7 @@ import StaticAssetsIcon from '../icon/StaticAssetsIcon';
 
 A full list of component-specific features can be viewed on each component's README.
```

Comment:
Looking at the deployment, my thought is we should update/merge the content on this page with the content on the component standards page: https://terra-ui-dep-proposed-r-0pjke2.herokuapp.com/about/terra-ui/contributing/component-standards

then we should drop the component-standards page. The feel redundant to me but may it's not bad